prow/cmd/deck/main.go

```diff
@@ -375,14 +379,14 @@ func main() {
 		}{
 			SpyglassEnabled: o.spyglass,
 			ReRunCreatesJob: o.rerunCreatesJob,
-			AllowAnyone:     cfg().Deck.RerunAuthConfig.AllowAnyone})
+			AllowAnyone:     authCfgGetter(nil).AllowAnyone})
```

Comment:
This effectively means that `AllowAnyone` can only be used as the global default, because otherwise the link in the button will redirect to the `/github-login` page, right?

Reply:
Yes, I see what you mean. I think omitting this field from the template entirely and performing the authorization check on the back-end is the appropriate remediation. I will verify this and update.

Reply:
There is no need for this check on the FE, since the BE has the info needed to verify rerun authorization. Updated.